tools/model_tools/src/openvino/model_zoo/download_engine/downloader.py

```diff
@@ -213,6 +213,12 @@ def _download_model(self, reporter, session_factory, model, known_precisions: se
 
         reporter.print_group_heading('Downloading {}', model.name)
 
+        model_unsupported_precisions = self.requested_precisions - model.precisions
+        if model_unsupported_precisions:
```

Comment:
If `self.requested_precisions & model.precisions` than you should't complain - at least something will be downloaded

Reply:
Done